vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,194 @@
+---
+title: "Setting up checklist"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Setting up checklist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Starting an R package from scratch
+
+### Prepare online setup
+
+1. Go to https://github.com and login.
+  Create a new public repository in the INBO organisation.
+  Use the package name as repository name.
+  Enter the repository name and optionally a description.
+  Don't add a `README`, `.gitignore` or `LICENSE`.
+
+### Prepare local setup
+
+1. Use `create_package()`.
+  You'll need to provide the package name, title, description and maintainer.
+  Please read the documentation of `create_package()` prior to running the function.
+  The function creates an RStudio project with a the package template in a folder with the name of the package.
+1. Open the newly created RStudio project.
+1. In the _Build pane_, select _More_ > _Configure Build Tools_.
+  Click the _Configure_ button and add a tick to _Install and Restart_.
+  All boxes except _Vignettes_ must be ticked.
+  Close the tow dialogue boxes by clicking the _OK_ buttons (once for each of box).
+1. Open the `DESCRIPTION` file.
+  Add a line `URL: https://inbo.github.io/_your_package_name_` where you replace `_your_package_name_` with the actual repository name.
+1. Open the `README.Rmd` file and update the content.
+  Knit the file and stage the `README.Rmd` and `README.md` files.
+1. Review and commit the changes into an initial commit.
+1. Add at least one function to the package
+1. _Build_ the package.
+1. Run `x <- checklist::check_package()`.
+  Fix any issues that arise during the checks.
+  Repeat the last two steps unless you end up with only a single new note about a _New submission_.
+1. Run `checklist::write_checklist(x)`.
```

Comment:
Do we have to run it always when we create a package, or only if we want to mark warnings or notes as allowed?  It would be good to specify when to run it, if this is not always needed.

Reply:
When your package is not on CRAN, you'll always get a note which you need to allow. It is very unlikely that you don't need to run `write_checklist()`.